README.md

```diff
@@ -31,7 +31,7 @@ The convinient methods in 0.1.0 have been removed, and now your application shou
 
 2 Reasons:
 
-* Each adal client should have a unique id representing an valid application registered in a tenant. The old methods borrowed the client-id of [azure-cli](https://github.com/Azure/azure-xplat-cli), which is never right. It is simple to register your application and get a client id. Many walkthroughs exist. You can follow [one of those](http://www.bradygaster.com/post/using-windows-azure-active-directory-to-authenticate-the-management-libraries). Though that involves C# client, but the flow, and particularly the wizard snapshots are the same with adal-python. Do check out if you are new to AAD.
+* Each adal client should have a unique id representing an valid application registered in a tenant. The old methods borrowed the client-id of [azure-cli](https://github.com/Azure/azure-xplat-cli), which is never right. It is simple to register your application and get a client id. Many walkthroughs exist. You can follow [one of those](https://docs.microsoft.com/en-us/azure/app-service-mobile/app-service-mobile-how-to-configure-active-directory-authentication). Do check out if you are new to AAD.
```

Comment:
Good catch for my copy-and-paste error! I mean to use [this App Registration link](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications) from the dev landing page instead.

As a side note, this entire paragraph was added 1 year ago, and its original purpose seems more like an explanation of a change after version 0.1.0. Now, it feels somewhat weird to have this section show up early in this README. Shall we demote this paragraph to the very end of README, in favor of drawing user's attention to the "Samples and Documentation" section instead?